compiler/damlc/util.bzl

```diff
@@ -9,3 +9,10 @@ load("@os_info//:os_info.bzl", "is_windows")
 # not going to work. We thus use the dynamically linked executable in the runfiles of damlc
 # and the tarball produced by package_app in the resources of damlc-dist.
 ghc_pkg = "@rules_haskell_ghc_windows_amd64//:bin/ghc-pkg.exe" if is_windows else "@ghc_nix//:lib/ghc-8.6.5/bin/ghc-pkg"
+
+DAML_LF_VERSIONS = [
```

Comment:
We have the same list in `//daml-lf/language:daml-lf.bzl`. Can we just kill this definiton?

Reply:
This is actually no as straightforward as it seems since the list in `daml-lf.bzl` uses `dev` instead of `1.dev`. I'll coordinate with @remyhaemmerle-da to see if we can change this to be `1.dev` instead. If so, I'll make a second PR.